### DISCUSSION TITLE
English (Shavian) is missing from app's language list

### DISCUSSION THREAD

**Discussion by JackknifedJuggernaut** [libre-tube/LibreTube#6782] [Ideas]
### Steps to reproduce

1. Open the app
2. Go to Settings → General
3. Open the language list

### Expected behavior

An option for the Shavian-script version of English titled “𐑦𐑙𐑜𐑤𐑦𐑖 (𐑖𐑱𐑝𐑾𐑯)” should appear in the language list, so that the corresponding locale can be selected as per [issue #6649](https://github.com/libre-tube/LibreTube/issues/6649)

### Actual behavior

The language list does not contain any entries corresponding to Shavian, hence the translation cannot be used in the app

### LibreTube version

0.26.0

### Android version

Android 15

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
I can't find the language code for it, not even sure if Android supports it.

If you find it, please create a PR modifying [https://github.com/libre-tube/LibreTube/blob/master/app/src/main/res/values-b%2Ben%2BShaw/strings.xml and ](https://github.com/libre-tube/LibreTube/blob/master/app/src/main/res/values/array.xml#L37) and https://github.com/libre-tube/LibreTube/blob/master/app/src/main/res/values/array.xml#L82.
